pkg/registry/pod/etcd/etcd_test.go

```diff
@@ -581,7 +581,7 @@ func TestEtcdUpdateNotScheduled(t *testing.T) {
 	key, _ := storage.KeyFunc(ctx, "foo")
 	key = etcdtest.AddPrefix(key)
 	opts := etcd.SetOptions{
-		TTL: time.Duration(1),
+		TTL: time.Duration(1) * time.Second,
```

Comment:
yeah, the fake client would need to change here, which is one of the many reasons for removing it entirely.